filestore/protoboards.go

```diff
@@ -0,0 +1,118 @@
+package filestore
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/influxdata/chronograf"
+)
+
+// ProtoboardExt is the the file extension searched for in the directory for protoboard files
+const ProtoboardExt = ".json"
```

Comment:
Any reason this needs to be exported?

Reply:
copied from layouts. happy to make private.